client/extensions/woocommerce/app/settings/payments/index.js

```diff
@@ -16,13 +21,23 @@ import SettingsPaymentsOnSite from './payments-on-site';
 class SettingsPayments extends Component {
 
 	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string,
+		} ),
 		className: PropTypes.string,
 	};
 
 	render() {
+		const { site, translate, className } = this.props;
+
+		const breadcrumbs = ( <span>
+			<a href={ getLink( '/store/:site/', site ) }>{ translate( 'Settings' ) }</a> &gt; { translate( 'Payments' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **43** times:
`translate( 'Payment' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).